src/main/java/org/apache/commons/math4/stat/descriptive/moment/Variance.java

```diff
@@ -315,7 +321,7 @@ public double evaluate(final double[] values, final int begin, final int length)
      */
     @Override
     public double evaluate(final double[] values, final double[] weights,
```

Comment:
Another method whose docs say that the first two parameters should have the same length, but you have not annotated with SameLen. I'm not going to add any more comments on it, but you should do a search through the codebase to make sure you've found them all.